articles/intro-to-C++-part01/index.md

```diff
@@ -0,0 +1,145 @@
+
+In this series of guides, will we go from basics to the more advanced levels.
+So let's jump straight into learning.
+
+### What do you need beforehand
+- A working PC or a Computer
+- An IDE
+
+The first question that comes to mind would be, what is an IDE? It is an Integrated Development Environment where you will write your C++ code and run it.
+To get started, you can download [Visual studio code](https://code.visualstudio.com/) as it is a simple and beginner-friendly text editor software.
+
+Let's dive into coding:
+
+### Hello World
+Let's write a Hello World program. We will break it down line by line:
+
+1. // This is a sample C++ Program
+2. #include<iostream>
+3. using namespace std;
+4. int main() {
+5. cout<<"Hello World"<<endl;
+6. return 0;
+7. }
+
+
+This program will print Hello World to the screen. Let's examine it line by line to see what it means.
+
+- Line 1 - **//This is a sample C++ Program**
+
+Any line of a program that begins with a "//" is a comment. The compiler will ignore this line.
+
+Comments are there to be read by people. You should always write comments for each major step of the algorithm to explain something weird or cool that is happening. That way, anyone working along with you or for you would be able to understand what you were trying to do.
+
+- Line 2 - **#include<iostream>**
+
+C++, by itself, cannot do input or output, so we add functionality to C++ by including libraries. For example, :<iostream> is a library that has commands for input from the keyboard and output to the screen.
+
+- Line 3 - **using namespace std;**
+
+It is something that you cannot avoid and have to write in every single program that you will write in your whole life. As defined by Danoja Das on [medium](https://medium.com/breaktheloop/why-using-namespace-std-is-used-after-including-iostream-dc5ae45db652), when we run a program to print something, “using namespace std” says if you find something that is not declared in the current scope go and check std. using namespace std; are used. It is because computer needs to know the code for the cout, cin functionalities and it needs to know which namespace are defined.
+
+- Line 4 - **int main ( ) {**
+
+The aforementioned line tells the compiler that this is the beginning of the main program. **{** means the start of the block of the program.
+
+- Line 5 - **cout<<"Hello World"<<endl;**
+
+**cout** is our "output to the screen" statement. **"endl"** takes you the next line. **";"** ends a C++ statement.
+
+- Line 6 - **return 0**
+
+This means that this is the end of our program.
+
+- Line 7 - **}**
+
+The line above is our closing bracket to end the program.
+
+Congratulations! You just wrote your first C++ program.
+
+Let's write another program that would read two numbers and output the sum of them.
+`  
```

Comment:
include `C` in your code fence so that it uses the proper syntax highlighting (see reference above)